Library/ViewModels/DashboardViewModel.swift

```diff
@@ -290,6 +293,28 @@ public final class DashboardViewModel: DashboardViewModelInputs, DashboardViewMo
 
     self.focusScreenReaderOnTitleView = self.viewWillAppearAnimatedProperty.signal.ignoreValues()
       .filter { AppEnvironment.current.isVoiceOverRunning() }
+
+    // MARK: - Tracking
+
+    self.viewWillAppearAnimatedProperty.signal.observeValues { _ in
+      AppEnvironment.current.ksrAnalytics.trackCreatorDasboardPageViewed()
+    }
+
+    _ = self.project
+      .takeWhen(self.switchToProjectProperty.signal)
+      .observeValues { project in
```

Comment:
We probably don't want to send the existing project with this tracking, but the project that is being switched _to_. try `takePairWhen`. Same thing for below signal for post update clicked.

Reply:
Also just be sure that these events actually get tracked in segment, because sometimes `_ = ` doesn't register an event at least in the past I've noticed sometimes' they can be flakey.

Reply:
hey just wanted to call out a change here when using `takePairWhen`, we use `_, param in` because we want the second parameter, not the first one like before. However, the catch here is that the second argument is not a project, so to find the `project` from the `param`, we see that in this view model all the projects were found through `fetchProjects` call and some filtering is done around line `155` to find the existing project from a param. Just copying and modifying slightly that logic applies to this tracking call too. Tested in Segment and it works as expected.

Reply:
Good question, I had to look at this carefully, I think you can't map directly on `projectsAndSelected`, because it goes to other signals, but `selectProjectPropertyOrFirst` is an intermediate signal that does tie in to `switchToProjectProperty`, which we use in our own `.map` function. It also includes the projects navigated to and from activities and message threads. If you want to track those other events as well, you can use `selectProjectPropertyOrFirst`, but you'd still need to re-write the map operation again, because the existing one lives inside the `projectsAndSelected`